spacy_loggers/mlflow.py

```diff
@@ -58,6 +66,15 @@ def setup_logger(
             mlflow.log_params({k.replace("@", ""): v for k, v in batch})
 
         console_log_step, console_finalize = console(nlp, stdout, stderr)
+        
+        if log_latest_dir:
+            latest_model = ModelDir()
+            
+        def log_model(path, name):
+            mlflow.log_artifacts(
+                path, 
+                name
+            )
```

Comment:
Similar: since this is just a wrapper without added functionality, let's use `mlflow.log_artifacts()` directly.